server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -208,26 +200,15 @@
   private File storageGroupSysDir;
 
   // manage seqFileList and unSeqFileList
-  private TsFileManagement tsFileManagement;
+  public TsFileManagement tsFileManagement;
```

Comment:
it would be better to set it as a private variable

Reply:
okay, I will solve it